### DISCUSSION TITLE
Github Action migration

### DISCUSSION THREAD

**Discussion by anbusampath** [spring-io/start.spring.io#1422] [General]
Curious to know the reason behind moving to Github Action from Concourse.

Migrate build to GitHub actions #1401
Remove Concourse build https://github.com/spring-io/start.spring.io/issues/1401 - https://github.com/spring-io/start.spring.io/commit/b7abb917ec539eb39828c3d7ecfc3a77bd53a38d

**Comment by wilkinsona** [comment]
The Spring team have CI running on numerous different platforms (at least Concourse, Jenkins, CircleCI, GitHub Actions) and we're trying to consolidate. GitHub Actions is the platform that best meets the team's needs right now.

**Reply by anbusampath** [reply]
Thanks for the reply.
